Apps/WebClient/src/ClientApp/src/components/dependentCard.vue

```diff
@@ -113,6 +108,11 @@ export default class DependentCardComponent extends Vue {
             });
     }
 
+    private removeDependent(): void {
+        // TODO:
```

Comment:
also, to make sure that we don't miss any thing as I don't see any other existing tasks for this // TODO:

Reply:
That's the story Sam is working on about deleting de dependant